tss-esapi/src/constants/property_tag.rs

```diff
@@ -56,6 +56,7 @@ pub enum PropertyTag {
     VendorCommands = TPM2_PT_VENDOR_COMMANDS,
     NvBufferMax = TPM2_PT_NV_BUFFER_MAX,
     Modes = TPM2_PT_MODES,
+    MaxCapBuffer = TPM2_PT_MAX_CAP_BUFFER,
```

Comment:
Do you know if there are any other fields like this that we might've missed? Asking because it's a fairly annoying (for users) change to make, so I'm not sure if we should just mark this enum as `non_exhaustive`

Reply:
Comparing with recent tpm2-tss, there are several new TPM2_CC constants, but they aren't used to construct an enum so there won't be any impact to users in adding those, either now or later.

I don't have a good feel for how likely this is to change in the future (presumably with future TCG spec revisions), so I don't have an opinion on whether or not to mark this as non_exhaustive.

Reply:
I am all for moving to new version of tpm2-tss but as @ionut-arm said there will be a rather big impact for the crate users. I do not think it is reasonable to try to bump the version in the major release 7 of the crate but I think it would be quite possible to do so in version 8.

Reply:
Maybe I'm missing something, but why does this imply needing to move to a new version of tpm2-tss?

Reply:
No there is no need to rise the version of tpm2-tss it was just me not reading the change nor the commit message properly.

Reply:
> but why does this imply needing to move to a new version of tpm2-tss?

Sorry, I think I didn't explain myself. Since this constant was added in version 2.4.0 and we mandate >= 2.3.3, it means there are some users for whom our crate builds properly (as they have 2.3.3), but they can't use this particular constant. This is combined with the fact that 2.3.3 is itself quite old, and we should move on a bit.

@Superhepper - why not bump the version in release 7?

Reply:
But the constant is defined in this crate and so is independent of the tpm2-tss version?

Reply:
Yes, I tried to track the history of the constants.rs file but I could not find if it at some point it had been generated from something or if it was something that was handwritten by us. Anyway that files is now a days updated by us manually. Adding the constant should work fine because it is a value that is returned from the TPM and bound to the TPM specification and not a specific version of tpm2-tss library. It was just a define for the value that was missing in the library and it got added in 2.4.0.

Reply:
@ionut-arm 
Well it is no problem to do it in major version 7 because it is still in alpha. I just thought it might be nice to give people a heads up before we do something like that.

Reply:
(About the constant) Ah, that's fair, I thought it was something the tpm2-tss library would also have to parse and understand, rather than just return from the TPM.

(About bumping the dependency version) Yeah, works for me to just wait for the next major release. It's a bit frustrating that there's still this gap between the spec and our support and I'm not sure how to bridge it, given that for every breaking change we have to bump the major. I'd lean towards just sticking with alpha for a while, releasing a 7.0.0, then whenever we get another breaking change we move to alpha for 8.0.0 and so on. It's nothing critical right now, it just constantly irks me